packages/portal/backend/src/server/common/GeneralRoutes.ts

```diff
@@ -208,6 +225,15 @@ export default class GeneralRoutes implements IREST {
         });
     }
 
+    public static generateDirectoryHtml(absolutePath: string, relativePath: string, baseUrl: string): string {
+        const directoryAddress: string = Config.getInstance().getProp(ConfigKey.publichostname) + baseUrl.replace(/\/$/, "");
```

Comment:
@braxtonhall 

There is another option available to you for serving static files in Restify. I have never mixed authentication business logic like Classy requires while serving static files in Restify, but it is probably possible by using middleware. I just wanted to make sure you knew about it, in case it helps.

```
server.get('/docs/current/*', restify.plugins.serveStatic({
  directory: './documentation/v1',
  default: 'index.html'
}));
```

Source: http://restify.com/docs/plugins-api/#servestatic